system/CodeIgniter.php

```diff
@@ -630,6 +630,25 @@ public function cachePage(Cache $config)
         return cache()->save($this->generateCacheName($config), serialize(['headers' => $headers, 'output' => $this->output]), static::$cacheTTL);
     }
 
+    /**
+     * Purges the cached page
+     *
+     * @param string     $uri    Uri
+     * @param Cache|null $config Config
+     */
+    public function purgeCachedPage(string $uri, ?Cache $config = null): bool
+    {
+        $uri = new URI($uri);
+
+        if (($config ?? new Cache())->cacheQueryString) {
```

Comment:
This should fall back on the shared instance. 
```suggestion
        if (($config ?? config('Cache'))->cacheQueryString) {
```